config/sprint_templates/tiny.json.erb

```diff
@@ -0,0 +1,66 @@
+{
+  "query": "<%= mutation %>",
+  "variables": {
+    "printer": "<%= printer %>",
+    "printRequest": {
+      "layouts": [
+        <% consumables.each_with_index do |consumable, i| %>
+        {
+          "labelSize": {
+            "width": 30,
+            "height": 12,
+            "displacement": 15
+          },
+          "barcodeFields": [
+            {
+              "x": 0,
+              "y": 5,
+              "cellWidth": 0.35,
+              "barcodeType": "datamatrix",
+              "value": "<%= consumable.barcode %>"
+            }
+          ],
+          "textFields": [
+            {
+              "x": 0,
+              "y": 3,
+              "value": "<%= consumable.barcode %>",
+              "font": "proportional",
+              "fontSize": 2.5
+            },
+            {
+              "x": 6,
+              "y": 6,
+              "value": "<%= consumable.consumable_type_name %>",
+              "font": "proportional",
+              "fontSize": 2
+            },
+            {
+              "x": 6,
+              "y": 9,
+              "value": "<%= consumable.batch_number %>",
+              "font": "proportional",
+              "fontSize": 2
+            },
+            {
+              "x": 17,
+              "y": 9,
+              "value": "<%= consumable.expiry_date %>",
+              "font": "proportional",
+              "fontSize": 2
+            },
+            {
+              "x": 6,
+              "y": 11,
+              "value": "#environment#",
```

Comment:
Does this `#environment#` need to be replaced somewhere?

Reply:
I'm planning on `tiny.json.erb` and any other being label templates being added via the Mixtio deployment project. This file is however used in some testing. (I should probably remove the `#environment#` part though as that's confusing)